lightly/embedding/embedding.py

```diff
@@ -56,33 +58,41 @@ class SelfSupervisedEmbedding(BaseEmbedding):
 
     """
 
-    def __init__(self,
-                 model: torch.nn.Module,
-                 criterion: torch.nn.Module,
-                 optimizer: torch.optim.Optimizer,
-                 dataloader: torch.utils.data.DataLoader,
-                 scheduler=None):
+    def __init__(
+        self,
+        model: torch.nn.Module,
+        criterion: torch.nn.Module,
+        optimizer: torch.optim.Optimizer,
+        dataloader: torch.utils.data.DataLoader,
+        scheduler=None,
+    ):
 
         super(SelfSupervisedEmbedding, self).__init__(
-            model, criterion, optimizer, dataloader, scheduler)
+            model, criterion, optimizer, dataloader, scheduler
+        )
 
-    def embed(self,
-              dataloader: torch.utils.data.DataLoader,
-              device: torch.device = None,
-              to_numpy: bool = True):
+    def embed(
+        self, dataloader: torch.utils.data.DataLoader, device: torch.device = None
+    ) -> Tuple[np.ndarray, np.ndarray, List[str]]:
         """Embeds images in a vector space.
 
         Args:
             dataloader:
                 A PyTorch dataloader.
             device:
                 Selected device (`cpu`, `cuda`, see PyTorch documentation)
-            to_numpy:
-                Whether to return the embeddings as numpy array.
 
         Returns:
-            A tuple consisting of a tensor or ndarray of embeddings
-            with shape n_images x num_ftrs and labels, fnames
+            The order of the return values is determined by the order of
```

Comment:
This sentence is the most important to you since you just worked on this issue but if someone else reads the docstrings I think it should say what it returns first and then go into detail about the order of the returned values (if they are even important). So something like:

```
Returns:
    Embeddings, labels, and filenames of the embedded images ordered by the
    samples in the dataset of the dataloader.
```